src/world/map.c

```diff
@@ -645,8 +645,8 @@ int map_is_location_owned(int x, int y, int z)
 {
 	rct_map_element *mapElement;
 
-	if (x < (256 * 32) && y < (256 * 32)) {
-		mapElement = map_get_surface_element_at(x / 32, y / 32);
+	mapElement = map_get_surface_element_at(x / 32, y / 32);
```

Comment:
The reason behind doing the check first was it would throw so many errors into the console.